src/makeInfinityAdder.js

```diff
@@ -1,12 +1,35 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
```

Comment:
The adder function should return itself when called with a number to allow for chaining calls. However, it currently does not return anything when called with a value.